doc/source/docs/api/geodataframe.rst

```diff
@@ -0,0 +1,87 @@
+============
+GeoDataFrame
+============
+.. currentmodule:: geopandas
+
+A ``GeoDataFrame`` is a tabular data structure that contains a column
+called ``geometry`` which contains a ``GeoSeries``.
```

Comment:
Content-wise nitpicks can be done as follow-ups as well, but "contains a column called geometry" is a bit confusing given that it doesn't need to be named that way